src/Modules/MotdModule/info.toml

```
@@ -0,0 +1,6 @@
+﻿[info]
+name = "MotdModule"
+title = "Message of the day"
+summary = "A module to display the motd to joining players"
+version = "1.0.0"
+author = "DasNeo"
```

```suggestion
author = "Evo"
```

For internal modules, we use "Evo" as the author.